react/wbooks/src/redux/auth/reducer.js

```diff
@@ -1,14 +1,21 @@
+/* eslint-disable no-case-declarations */
+import api from '../../app/config/api';
+
 import { actions } from './actions';
 
 const initialState = {
-  accessToken: null
+  accessToken: localStorage.getItem('accessToken')
 };
 
 function reducer(state = initialState, action) {
   switch (action.type) {
     case actions.SET_ACCESS_TOKEN:
-      return { ...state, accessToken: action.payload };
+      const accessToken = action.payload;
+      api.setHeader('Authorization', accessToken);
+      localStorage.setItem('accessToken', accessToken);
+      return { ...state, accessToken };
     case actions.REMOVE_ACCESS_TOKEN:
+      localStorage.removeItem('accessToken');
       return { ...state, accessToken: null };
```

Comment:
wait, the reducer work is just to reduce, those setHeaders and others should be applied in an action